tests/unit-test/test_serializer.c

```diff
@@ -12,6 +12,14 @@
 #include "accelerator/core/serializer/serializer.h"
 #include "tests/test_define.h"
 
+static void gen_rand_trytes(int len, tryte_t* trytes) {
+  const char tryte_alpahbet[] = "NOPQRSTUVWXYZ9ABCDEFGHIJKLM";
+
+  for (int i = 0; i < len; i++) {
+    trytes[i] = tryte_alpahbet[rand() % 27];
+  }
+}
```

Comment:
It this possible to use the same `gen_rand_trytes()` without repeated implementation in other files? 
Such as ` tests/api/common.c` has its own `gen_rand_trytes()` but the implementation is the same as ` tests/unit-test/test_serializer.c`.

Reply:
I will implement this function under `tests/`